modules/porous_flow/paper.md

```diff
@@ -23,28 +23,72 @@ date: 13 February 2020
+- Multiphase modelling, such as the simulation of CO$_{2}$ sequestration [@green2018; @lima], H$_{2}$ storage, or methane flows around coal mines [@c26050; @c25065].  These simulations involve the interaction and flow of at least two fluid phases (e.g. gas and water).  Changes in temperature often impact these flows, and realistic equations of state for the fluids involved are necessary.  For instance, CO$_{2}$ sequestration typically involves pumping high-pressure, cold CO$_{2}$ into a warmer aquifer filled with brine: the CO$_{2}$ dissolves into the brine, cooling it and changing its density and flow characteristics.  In many of these models, the altered pressure and temperature can potentially cause mechanical failure of the subsurface, and hence the creation of new flow paths and microseismicity [@lima].
 
 - Understanding mineralization [@heather] in the subsurface often involves fluid flow, heat flow and coupled 3D deformation, simulating over millennia of real time and using realistic, and therefore complicated, basin-scale geology.
 
 Other situations where ``PorousFlow`` could be used include oil and gas production, nuclear waste repositories and chemical leaching.
 
-In many of these cases it is not enough to simply solve simplistic, traditional flow and transport physics.  Coupling with other physics is necessary: solid mechanical deformations and stresses can be important; geochemistry can alter the flow characteristics of the subsurface; high-precision equations of state are required; the evolution of fluid components (tracers, pollutants, reactants) is of interest; unsaturated physics is crucial.  The flow and transport often impacts the subsurface structure, and conversely, the effects of these structure-changes impact the flow and transport.
+In many of these cases it is not enough to only solve simplistic, traditional flow and transport physics.  Coupling with other physics is necessary: solid mechanical deformations and stresses can be important; geochemistry can alter the flow characteristics of the subsurface; high-precision equations of state are required; the evolution of fluid components (tracers, pollutants, reactants) is of interest; unsaturated physics is crucial.  The flow and transport often impacts the subsurface structure, and conversely, the effects of these structure-changes impact the flow and transport.
```

Comment:
Maybe remove the word simplistic here altogether?

Reply:
Thanks, done